### DISCUSSION TITLE
Adapter V5 - problems flashing firmware

### DISCUSSION THREAD

**Discussion by physman38** [john30/ebusd#1003] [Q&A]
Hi all,

I received the adapter v5 a while ago but now had time to try and followed the instruction.
First struggle I had was with the WebSerial of Chrome - what should I enter where?
I also pressed the little button when connecting to the USB port - the blue light flashed but stopped. Repeating it does not bring it back to flashing.
Than I had once seen the webserial (forgot where I found it) but I saw no options like in the "get started" with the firmwares to choose etc.

Does anyone have an idea or can help?
Do I need to do everything from scratch but how?

Thanks for any detailed help....

**Comment by HeikoGr** [comment]
You can select the desired firmware on this page: https://adapter.ebusd.eu/v5/firmware.en.html
If you have an propper USB cable with data lines (some cheap USB cables only suport power) Chrome should ask for permission to access the serial port on the upper left corner after clicking "connect".

maybe you need some additional drivers.
Hint: Just select the latest firmware and press the button, you may get more help from the popup if no serial port is available.
